packages/hint/docs/contributor-guide/how-to/hint.md

```diff
@@ -93,15 +93,15 @@ with the website and report errors.
 To report an error, the hint has to do the following:
 
 ```ts
-await context.report(resource, message, { element: element });
+context.report(resource, message, { element: element });
 ```
 
-* `context.report()` is an asynchronous method, you should always `await`.
+* `context.report()` is a synchronous method, you don't need to `await`.
```

Comment:
Nit: We can probably remove this line, I think we needed before to make it clear but being sync is the expected thing now.